modin/pandas/test/test_series.py

```diff
@@ -2086,19 +2088,37 @@ def test_pow(data):
     inter_df_math_helper(modin_series, pandas_series, "pow")
 
 
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_prod(data):
-    modin_series, pandas_series = create_test_series(data)
-    # Wrap in Series to test almost_equal because of overflow
-    df_equals(pd.Series([modin_series.prod()]), pandas.Series([pandas_series.prod()]))
-
-
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_product(data):
-    modin_series, pandas_series = create_test_series(data)
-    # Wrap in Series to test almost_equal because of overflow
-    df_equals(
-        pd.Series([modin_series.product()]), pandas.Series([pandas_series.product()])
+@pytest.mark.parametrize(
+    "data",
+    test_data_values + test_data_small_values,
+    ids=test_data_keys + test_data_small_keys,
+)
+@pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
+@pytest.mark.parametrize(
+    "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
+)
+@pytest.mark.parametrize(
+    "numeric_only",
+    [
+        None,
+        False,
+        pytest.param(
+            True,
+            marks=pytest.mark.xfail(reason="numeric_only not implemented for Series"),
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "min_count", int_arg_values, ids=arg_keys("min_count", int_arg_keys)
+)
+def test_prod(data, axis, skipna, numeric_only, min_count):
+    eval_general(
+        *create_test_series(data),
+        lambda df, *args, **kwargs: type(df)([df.prod(*args, **kwargs)]),
```

Comment:
You should test `product` also.

Reply:
`product` is just an alias for `prod`, I don't think we want to run 2000+ extra tests to test the same things

Reply:
Is there a chance that this behavior could change in the future?

Reply:
well, there is always a chance of everything...

but for now, modin and pandas have that line in their `DataFrame`/`Series` classes:
```python
product = prod
```

Reply:
Well, but we can check that `product` is alias of `prod`. When and if it doesn't work that way, we can add tests. What about that way?

Reply:
done it